submissions/chabuChabu/php/LoopsAndNumber.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+//Display every number between 100 and 150
+
+for($i = 100;$i < 151;$i++){
+ echo $i."<br>";
+}
+
+//Display even numbers between 0 and 50
+
+$x = 0;
+
+while($x < 50) {
+    $x += 2;
```

Comment:
This approach is ok, but not a good way of knowing if a number is an even number or not. You will have to do a rework